desktop-web-app/web-app-vue/src/App.vue

```diff
@@ -0,0 +1,28 @@
+/* Copyright (c) 2025 TeamDev Permission is hereby granted, free of charge, to any person obtaining
```

Comment:
This looks strange. Can we use a regular format with * at the start of the line?